ui/app/AppLayouts/Profile/views/LanguageView.qml

```diff
@@ -12,42 +14,127 @@ import "../popups"
 import "../stores"
 
 Item {
-    id: languageContainer
-    Layout.fillHeight: true
-    Layout.fillWidth: true
-    clip: true
+    id: root
 
     property LanguageStore languageStore
     property int profileContentWidth
-    property Component languagePopup: LanguageModal {
-        languageStore: languageContainer.languageStore
+
+    QtObject {
+        id: d
```

Comment:
can we rename this to something more meaningful like _internal or something?

Reply:
I would like to keep as it is because of [QML Convention from Qt - Private properties](https://doc.qt.io/archives/qt-4.8/qml-coding-conventions.html) and discussion [here](https://github.com/status-im/StatusQ/pull/594#discussion_r832494124). WDYT?